### PR TITLE
open-cluster-management/backlog#4558 -- dev review (1)

### PR DESCRIPTION
**Pull request by swopebe**
@mikeshng this is my first draft. I still have lots of questions and I have not addressed everything, but if you want or have time to look at it before our meeting--we may be able to clear some things up.

I will be working on it today, too.

### CODE REVIEW

**Comment by mikeshng** [comment]
Hi Brandi, this is amazing work. I added my commit here: https://github.com/open-cluster-management/rhacm-docs/pull/1373/commits/f80cba3ba68138032c1920fb99ed06ee8b69f84d

**Comment by mikeshng** [comment]
Added another commit: https://github.com/open-cluster-management/rhacm-docs/pull/1373/commits/57bd059e1b8f40b7f22f25b51a2c317456c9fc47 to hopefully clarify the difference between
AnsibleJob - The Kubernetes CR that lives on OCP/k8s
Ansible Tower job - The actual job instance that is created on the Ansible Tower side.

**Comment by swopebe** [comment]
oh wow @mikeshng , thank you--unexpected. And thank you for the points. Given that we can do the rest over this PR, I will cancel today's meeting.

File: manage_applications/ansible_config.adoc [hunk 1]
```diff
@@ -1,91 +1,105 @@
...
+apiVersion: v1
+kind: Secret
+metadata:
+  name: toweraccess
+  namespace: same-as-subscription
+type: Opaque
+stringData:
+  token: ansible-tower-api-token
+  host: https://ansible-tower-host-url
+----
```
Comment (swopebe) [inline manage_applications/ansible_config.adoc:86]: @mikeshng do we want to give them an apply command here?

**Comment by swopebe** [comment]
I am going to merge since we did quite a bit here and get the last two lines I have taken care of in a new PR with a peer review.

File: manage_applications/ansible_config.adoc (hunk 1 above)
Comment (mikeshng) [reply manage_applications/ansible_config.adoc:86]: the command is the `oc apply -f`

**Pull request merged**